assets/airports/lrop.json

```diff
@@ -0,0 +1,1409 @@
+                "LROP08L": [["^OP883", "A40+|S250-"]],
+                "LROP08R": [["^OP884", "A40+|S250-"]]
+            },
+            "body": ["OP889", "USIRO"],
+            "exitPoints": {
+                "POLUN": ["POLUN"]
+            },
+            "draw": [
+                ["OP884", "OP889", "USIRO", "POLUN"],
+                ["OP883", "OP889", "USIRO", "POLUN"]
+            ]
+        },
+        "POLUN1M": {
+            "icao": "POLUN1M",
+            "name": "Polun One Mike",
+            "rwy": {
+                "LROP26L": [["OP666", "A40+"]],
+                "LROP26R": [["OP665", "A40+"]]
+            },
+            "body": ["SISGU"],
+            "exitPoints": {
+                "POLUN": ["POLUN"]
+            },
+            "draw": [
+                ["OP665", "SISGU", "POLUN"],
+                ["OP666", "SISGU", "POLUN"]
+            ]
+        },
+        "SOKRU1K": {
+            "icao": "SOKRU1K",
+            "name": "Sokru One Kilo",
+            "rwy": {
+                "LROP08L": [["^OP881", "A09+|S220-"]],
+                "LROP08R": [["^OP882", "S220-"]]
+            },
+            "body": ["OP890", ["OP891", "A45+"], "GOMDI"],
+            "exitPoints": {
+                "SOKRU": ["SOKRU"]
+            },
+            "draw": [
+                ["OP881", "OP890", "OP891", "GOMDI", "SOKRU"],
+                ["OP882", "OP890", "OP891", "GOMDI", "SOKRU"]
+            ]
+        },
+        "SOKRU1M": {
+            "icao": "SOKRU1M",
+            "name": "Sokru One Mike",
+            "rwy": {
+                "LROP26L": [["^OP664", "S250-"]],
+                "LROP26R": [["^OP663", "S250-"]]
+            },
+            "body": ["UTOKI"],
+            "exitPoints": {
+                "SOKRU": ["SOKRU"]
+            },
+            "draw": [
+                ["OP663", "UTOKI", "SOKRU"],
+                ["OP664", "UTOKI", "SOKRU"]
+            ]
+        }
+    },
+    "stars": {
+        "DENAK1U" : {
+            "icao": "DENAK1U",
+            "name": "Denak One Uniform",
+            "entryPoints": {
+                "DENAK": [["DENAK", "A170-|A110+"]]
+            },
+            "body": [["IPRAS", "A70+"], ["INDOR", "A70+|S240-"], ["OP087", "A70-|S230-"], "OP086", "OP085", ["OP084", "A35+|S220-"], ["OP083", "A35-|S220-"], "OP082", "OP081"],
+            "rwy": {
+                "LROP08L": [["OBELA", "A25+|S210-"], "_FAF08L", "_THR08L"],
+                "LROP08R": [["DILAS", "A25+|S210-"], "_FAF08R", "_THR08R"]
+            },
+            "draw": [
+                ["DENAK", "IPRAS", "INDOR", "OP087", "OP086", "OP085", "OP084", "OP083", "OP082", "OP081", "DILAS"],
+                ["DENAK", "IPRAS", "INDOR", "OP087", "OP086", "OP085", "OP084", "OP083", "OP082", "OP081", "OBELA"]
+            ]
+        },
+        "DENAK2X" : {
+            "icao": "DENAK2X",
+            "name": "Denak Two Xray",
+            "entryPoints": {
+                "DENAK": [["DENAK", "A170-|A110+"]]
```

Comment:
No restriction at DENAK

Reply:
regarding the DENAK limitation (and all others) everything outside the TMA is uncontrolled airspace up to FL105 (i've updated the charts in the first post), so each airway has a minimum level of followed strictly, otherwise, they have to be FL110 or above. for example: at TOSVI one should be at FL110 if landing on 08L/R or at FL 170 if landing on 26L/R

 I'll check them again to see if they match the charts

![image](https://user-images.githubusercontent.com/1264087/38453302-ffbe265a-3a5b-11e8-931d-fa4ba6668be0.png)

Reply:
Fair enough, disregard then